cineast-api/src/main/java/org/vitrivr/cineast/api/APIEndpoint.java

```diff
@@ -352,6 +352,11 @@ private void registerRestOperations() {
         new FindSegmentsByIdGetHandler(),
         new FindSegmentsByObjectIdGetHandler(),
         new FindSegmentSimilarPostHandler(retrievalLogic),
+        new FindSegmentFeaturesGetHandler(),
+        new FindSegmentTagsGetHandler(),
+        new FindSegmentCaptionsGetHandler(),
+        new FindSegmentOCRGetHandler(),
```

Comment:
Currently, "find/segment/asr/by/id/:" and "find/segment/ocr/by/id/:" can be queried directly in the API. 
If we were to change the handling of those features to one handler, both would always be queried together and not as easily distinguishable as they are now. We would then need a new approach, where each query and response are typed either ASR or OCR. We like our solution because it allows for individual API calls per feature, as we do with tags and captions as well.